app/webpack.production.js

```diff
@@ -4,31 +4,13 @@ const common = require('./webpack.common')
 
 const webpack = require('webpack')
 const merge = require('webpack-merge')
-const ExtractTextPlugin = require('extract-text-webpack-plugin')
-const BabelPlugin = require('babel-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer')
 
 const config = {
+  mode: 'production',
   devtool: 'source-map',
-  plugins: [
```

Comment:
Using `mode: 'production'` here suggests that we don't need to do this follow up work: https://webpack.js.org/concepts/mode/#mode-production

Reply:
What do you think about using [`optimization.minimizer`](https://webpack.js.org/configuration/optimization/#optimization-minimizer) to disable Uglify and use Babel-Minify so that we can take advantage of the shorter constructs supported by ES6+?

Reply:
Maybe? At first glance the `main.js` file is ~1KB smaller and `renderer.js` is ~3KB smaller, so the defaults aren't worse in that metric than what we were doing before. I'll dig more into it next week.

Reply:
Digging into this some more, it looks like [`uglify-es`](https://www.npmjs.com/package/uglify-es) is the equivalent ES6+ support for UglifyJS, and after going down the path of installing `uglify-es-webpack-plugin` I instead got this warning/recommendation:

```
warning uglify-es-webpack-plugin@0.10.0: webpack-contrib/uglifyjs-webpack-plugin now has uglify-es support, featuring tests and far better implementation
```

The version we now have inside `yarn.lock` also contains `uglify-es`:

```
uglifyjs-webpack-plugin@^1.2.4:
  version "1.2.5"
  resolved "https://registry.yarnpkg.com/uglifyjs-webpack-plugin/-/uglifyjs-webpack-plugin-1.2.5.tgz#2ef8387c8f1a903ec5e44fa36f9f3cbdcea67641"
  dependencies:
    cacache "^10.0.4"
    find-cache-dir "^1.0.0"
    schema-utils "^0.4.5"
    serialize-javascript "^1.4.0"
    source-map "^0.6.1"
    uglify-es "^3.3.4"
    webpack-sources "^1.1.0"
    worker-farm "^1.5.2"
```

So I don't really see the value of switching back to `babel-minify` if Uglify has caught up enough and we still have the tree-shaking and dead code elimination that motivated #1225 in the first place.